### DISCUSSION TITLE
Failed to load image issue

### DISCUSSION THREAD

**Discussion by EmilienAlvarez** [biigle/core#588] [General]
Hello,
I recently tried to use Biigle.de on a Mac computer but got the error message "Failed to load image 3057780!" when trying to view large images for labeling. The error occurred with the Safari, Chrome and Firefox browsers on the Mac, but not when I switched to Windows with another computer.

A month ago, the application worked perfectly on the Mac. Is there a new problem with Mac or is it perhaps due to something else?

Thank you for your understanding,

Emilien

**Comment by mzur** [comment]
@dlangenk can you please check this out with your MacBook?

**Reply by dlangenk** [reply]
For me it works in Safari/Firefox/Chrome. But likely it has nothing to do with the operating system but rather the computer or OS settings.

**Comment by dlangenk** [comment]
@EmilienAlvarez Could you try in private/incognito mode (In Chrome it's Command + Shift + N) and see if it works?
